src/core/Akka/Actor/ActorPath.cs

```diff
@@ -254,8 +234,7 @@ public bool Equals(ActorPath other)
             var a = path;
             foreach (string element in name)
             {
-                if(!string.IsNullOrEmpty(element))
-                    a = a / element;
+                a = a / element;
```

Comment:
Removing the `if(!string.IsNullOrEmpty(element))` line here is what caused the `ClusterSingletonManagerStartupSpec` to fail - the `ActorSelection`'s used by the `ClusterSingletonProxy` can start with an empty first element due to the way their `split` function is implemented without this. Going to run a regression test on the build server real quick, but this looks like it.